FirebaseCombineSwift/README.md

```diff
@@ -58,4 +58,54 @@ end
     .replaceError(with: "(unable to sign in anonymously)")
     .assign(to: \.uid, on: self)
     .store(in: &cancellables)
-```
\ No newline at end of file
+```
+#### Sign in with a given 3rd-party credentials 
+
+
+In the `sign(_:didSignInFor:withError:)` method, get a Google ID token and Google access token from the GIDAuthentication object and exchange them for a Firebase credential. Then, Asynchronously sign in to Firebase with the given tokens pair:
+
+```swift 
+    func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error?) {
+       // ...
```

Comment:
Google indentation style is 2 spaces. Can you update the entire snippet accordingly?